webpack.config.js

```diff
@@ -55,4 +57,13 @@ module.exports = {
         new webpack.optimize.UglifyJsPlugin(),
         new webpack.optimize.DedupePlugin(),
     ],
+    module: {
+        loaders: [
+            {
+                test: /\.js$/,
+                exclude: /(node_modules|bower_components|javascripts-legacy|components|vendor)/,
```

Comment:
That's odd - I had something similar set up in the [preact prototype webpack](https://github.com/guardian/frontend-future/blob/master/prototypes/preact/webpack.config.dev-ssr.js#L22)  - but no point messing about with it if it works!